go.mod

```diff
@@ -43,7 +43,7 @@ require (
 	github.com/mroth/weightedrand/v2 v2.1.0
 	github.com/openacid/low v0.1.21
 	github.com/oschwald/maxminddb-golang v1.12.0 // lastest version compatible with golang1.20
-	github.com/saba-futai/sudoku v0.0.1-e
+	github.com/saba-futai/sudoku v0.0.1-f
```

Comment:
should remove
https://github.com/SUDOKU-ASCII/sudoku/blob/v0.0.1-f/pkg/obfs/sudoku/table.go#L143